config/locales/javascript/javascript.en.yml

```diff
@@ -70,9 +70,17 @@ en:
           other: "<%= count %>ms"
         unknown: "unknown"
         not_available: "not available"
+        total:
+          one: "There is only one known pod."
+          other: "There are <%= count %> known pods in total."
         unchecked:
           one: "There is still one pod that hasn't been checked at all."
           other: "There are still <%= count %> pods that haven't been checked at all."
+        active:
+          one: "One pod was active."
+          other: "<%= count %> pods were active recently."
+        none_active: "None of them were active."
+        all_active: "All of them were active."
```

Comment:
The other times still miss the "recently". I mean all pods in the list were active at some point in time, but maybe not recently, so that's the important thing here.

Reply:
Oh.